examples/log/src/main.rs

```diff
@@ -222,7 +222,6 @@ fn main() {
             activity_timeout: ViewDelta::new(10),
             skip_timeout: ViewDelta::new(5),
             fetch_concurrent: 32,
```

Comment:
is there a way to expose a `hint()` from the limited sender? That would allow us to provide a dynamic concurrency recommendation (`online * RL`)?

Reply:
This would be a great followup issue 👁️